lib/api/route/messages.dart

```diff
@@ -327,6 +327,51 @@ class UploadFileResult {
   Map<String, dynamic> toJson() => _$UploadFileResultToJson(this);
 }
 
+/// Get a temporary, authless partial URL to a realm-uploaded file.
+///
+/// The URL returned allows a file to be viewed without requiring authentication,
+/// but it doesn't include secrets like the API key. This URL remains valid for
+/// 60 seconds.
+///
+/// This endpoint is documented in the OpenAPI description:
+/// https://github.com/zulip/zulip/blob/main/zerver/openapi/zulip.yaml
+/// under the name `get_file_temporary_url`.
```

Comment:
```suggestion
/// under the name `get-file-temporary-url`.
```
I couldn't find a match of `get_file_temporary_url` searching in [zulip.yaml](https://github.com/zulip/zulip/blob/main/zerver/openapi/zulip.yaml), but searching `get-file-temporary-url` has a match.